<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Pobieranie pliku w protokole HTTP</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="mywindow.qrc" >:/images/qt.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="labelHTTP" >
     <property name="text" >
      <string>Podaj adres:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLineEdit" name="lineEdit" >
     <property name="toolTip" >
      <string>Adres URI zasobu (pliku)</string>
     </property>
     <property name="text" >
      <string>http://</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="pushButtonGet" >
     <property name="toolTip" >
      <string>Pobierz wsskazany plik</string>
     </property>
     <property name="text" >
      <string>Pobierz</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="pushButtonQuit" >
     <property name="toolTip" >
      <string>Zakończenie działania programu</string>
     </property>
     <property name="text" >
      <string>Wyjście</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mywindow.qrc" />
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
